pkg/stacker/convert.go

```diff
@@ -166,12 +166,21 @@ func (c *Converter) convertCommand(cmd *Command) error {
 		log.Infof("EXPOSE directive found - ports:%v", cmd.Value)
 		return nil
 	case "env":
-		if len(cmd.Value) != 2 {
+		key := ""
+		val := ""
+
+		if len(cmd.Value) == 1 && strings.Contains(cmd.Value[0], "=") {
+			parts := strings.SplitN(cmd.Value[0], "=", 2)
```

Comment:
does this also "strip" leading/trailing whitespace?

Reply:
```
unable to parse file Dockerfile
   convert failed with err:&{[{{%!!(MISSING)e(int=10) %!!(MISSING)e(int=0)} {%!!(MISSING)e(int=10) %!!(MISSING)e(int=0)}}] %!!(MISSING)e(*errors.fundamental=&{Syntax error - can't find = in "1234567890". Must be of the form: name=value 0xc000012f30})}
   2023/08/04 22:30:04 conversion failed: &{[{{%!e(int=10) %!e(int=0)} {%!e(int=10) %!e(int=0)}}] %!e(*errors.fundamental=&{Syntax error - can't find = in "1234567890". Must be of the form: name=value 0xc000012f30})}
   error: exit status 1
   ```
   
   Seems like it isn't a valid Dockerfile itself (?)

Reply:
```
docker build --tag test/test:dev -f Dockerfile .
[+] Building 0.2s (2/2) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                     0.1s
 => => transferring dockerfile: 470B                                                                                                                                                                                                                     0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                                                                                                                          0.0s
failed to solve with frontend dockerfile.v0: failed to create LLB definition: Syntax error - can't find = in "1234567890". Must be of the form: name=value